components/mfa/components/confirm.jsx

```diff
@@ -13,6 +13,7 @@ export default class Confirm extends React.Component {
     constructor(props) {
         super(props);
 
+        this.submit = this.submit.bind(this);
```

Comment:
The pattern that we use for solve this problem is use arrow functions in the definition, for example `submit = (e) => {`. Some files (like this) aren't migrated to this pattern yet, so if you can change to it, and change the onKeyPress too, I think would be better. :)

Reply:
Ah, excellent feedback! I wasn't sure which pattern was preferred, and I agree that the arrow functions better captures the intent (since it's clearly easy to miss the pre-flight bind in the constructor). I've made the requested changes.